game-service/src/main/java/org/libertybikes/game/core/GameBoard.java

```diff
@@ -60,7 +60,7 @@ public boolean addObstacle(MovingObstacle o) {
         return verifyObstacle(o) ? movingObstacles.add(o) : false;
     }
 
-    public Player addPlayer(String playerId, String playerName) {
+    public Player addPlayer(String playerId, String playerName, int[][] playerPositions) {
```

Comment:
It would be cleaner if adding a player/ai to a game didn't require knowledge of where that player should go.  Instead, the maps should be able to define where each player slot is located and fill players into the appropriate locations.  I think we can accomplish this by randomly selecting a game map in the `GameBoard` ctor, and then storing the `GameMap` instance as a field on `GameBoard` and then checking `gameMap.getPlayerPositions()` in addPlayer/addAI